Petko/Zad3.cpp

```diff
@@ -0,0 +1,70 @@
+// ConsoleApplication1.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+enum class Course
+{
+	first,
+	second,
+	third,
+	fourth
+};
+void printCourse(const Course& st)
+{
+	switch (st)
+	{
+	case Course::first:cout << "first"; break;
+	case Course::second:cout << "second"; break;
+	case Course::third:cout << "third"; break;
+	case Course::fourth:cout << "fourth"; break;
+	}
+}
+struct Student
+{
+	double averageCourseMark;
+	int fN;
+	Course student;
+	char name[17];
+};
+void printStudent(const Student& toPrint)
+{
+	cout << toPrint.averageCourseMark << toPrint.fN << toPrint.name;
+	printCourse(toPrint.student);
+}
+bool initStudent(Student& toInit, char name[16], int facNumber, Course currentCourse, double grade)
+{
+	cin >> grade;

```

Comment:
It'll be great to add some prompts here - otherwise it's a bit of a guessing game this one